CHANGELOG.md

```diff
@@ -22,8 +22,7 @@ and this project adheres to
 - `cryptoPrefetcher()` to throw when `bufSize` is not a multiple of
   `valueSize`.
 - `MemoryWritable` internal buffer size is now limited to 8 MB by default.
-- Signature of `callerFilepath()` to allow passing `RegExp` as depth to be used
-  for filtering of stack frames.
+- `sortComparePriority` function contract changed.
```

Comment:
Elaborate on the change, it isn't clear in what way the contract was changed from reading that.

Reply:
Something like this?

- `sortComparePriority` function contract changed: now it accepts only one argument - `priority` and returns a function, which accepts 2 arguments - filenames. E.g. `sortComparePriority(priority)(file1, file2)`

Reply:
First of all, there is no need to repeat the word "changed", it is already specified in the category header above, so just start with

> Contract of `sortComparePriority()` 

So that when its read after the header it combines into:

> Changed contract of `sortComparePriority()`

Secondly, no need to explicitly state that it is a function or a method, the parentheses after the identifier name is a common style that is used to refer to functions/methods in the documentation.

And finally, in your description, you are referring to "filenames", that were never mentioned before neither in the code nor in the documentation of this function. I don't think that the usage of the resulting function is restricted to file names only, so there is probably no need to specify that constraint.

Summarizing the points above, I'd go for something like the following:
- Contract of `sortComparePriority()` to return a function capturing the first argument and accepting the other two.

I'd prefer to get a second opinion on the specific wording of the change though, maybe @lundibundi can chime in.